### ISSUE TITLE
Daily stacked and spent summary notifications

### ISSUE THREAD

**Issue by huumn** [stackernews/stacker.news#1751]
This has been requested many times.

Stackers want an overview of how much they earned relative to spent. 

This shouldn't be too hard to do and can be pretty minimal to start. We'd just grab `user_stats` for the prior day and make sure to add any rewards that were just given out (at midnight), then the notification could say "you stacked X sats and spent X sats."

**Comment by huumn** [comment]
I personally want this so I can try to spend as much as I stacked every day.

**Comment by m0wer** [comment]
Does this still need work? I've implemented it before realizing there already was https://github.com/stackernews/stacker.news/pull/1756 🤦

**Comment by m0wer** [comment]
Thanks @Soxasora ! What's a “seed”? Seed funding round or something else?

Here is the code: https://github.com/stackernews/stacker.news/pull/2132

I would need some help or ideas on a simple test scenario. I've tried creating an email account and zapping and then change the Docker host time to one day in the future but that wasn't enough to get something on the past day for `user_stats_days`. I might be missing something. I tried running the job `materializedViewRefresher` and then the new `dailyStackedNotification`.

**Comment by huumn** [comment]
Please hold off on this! I'm currently refactoring related code to make this a lot easier. I'm adding the `blocked` label in the meantime.

**Comment by m0wer** [comment]
Should this be unblocked?

**Comment by ekzyis** [comment]
No, this is still blocked by https://github.com/stackernews/stacker.news/pull/2084
